preview/Dockerfile

```diff
@@ -1,28 +1,34 @@
-FROM buildpack-deps:trusty-scm
+FROM buildpack-deps:jessie-scm
```

Comment:
Have we considered using `jessie-curl` instead?  That image appears to be 42 MB smaller.  According to the [buildpack-deps repo](https://hub.docker.com/r/library/buildpack-deps/), the scm images add "various source control management tools"; I'm wondering how important those tools are for the base .NET Docker image.